components/x11/glw/Makefile

```diff
@@ -35,10 +36,12 @@ PATCH_LEVEL=0
 
 LDFLAGS= -L$(X11_SERVERLIBS_DIR) -R$(X11_SERVERLIBS_DIR)
 
-CONFIGURE_OPTIONS += --enable-motif
+#CONFIGURE_OPTIONS += --enable-motif
+REQUIRED_PACKAGES += developer/build/autoconf/xorg-macros
+REQUIRED_PACKAGES += x11/header/x11-protocols
 
 # Auto-generated dependencies
-REQUIRED_PACKAGES += library/motif
+# REQUIRED_PACKAGES += library/motif
 REQUIRED_PACKAGES += system/library
```

Comment:
Did you really run gmake REQUIRED_PACKAGES to create this?

Reply:
that was manually.
i now did use gmake REQUIRED_PACKAGES